### ISSUE TITLE
Experimental technique

### ISSUE THREAD

**Issue by RKrahl** [rdawg-pidinst/schema#67]
The [ExPaNDS](https://expands.eu/) project has developed [PaNET – The Photon and Neutron Experimental Techniques Ontology](https://github.com/ExPaNDS-eu/ExPaNDS-experimental-techniques-ontology). I assume other (experiment performing) scientific communities might have similar vocabularies for their techniques. Obviously, it would be interesting to link an instrument with the experimental technique(s) it has been designed to use. Surprisingly enough, there doesn't seem to be a proper place in our schema to do that.

Experimental techniques seem to be somewhat complementary to `MeasuredVariable`: the latter is the physical observable that the instrument records. The technique goes way beyond that, it describes the physical phenomenon that the instrument makes use of or the measurement protocol it performs in order to make an observation. `MeasuredVariable` is fair enough for simple instruments like temperature sensors. But it is too limited to describe complex instruments such as an experimental station for x-ray absorption spectroscopy.

Interestingly enough, it is possible to include the link from the instrument to the experimental technique in the DataCite schema: the `subject` property in DataCite is mostly used to just add keywords. But using its subproperties, it is versatile enough to also establish the link to a PaNET term. I did that excercise for one of our [instrument DOIs](https://doi.org/10.5442/NI000001) (see the [DataCite XML metadata](https://api.datacite.org/application/vnd.datacite.datacite+xml/10.5442/ni000001)):
```xml
  <subjects>
    <subject>BER II</subject>
    <subject>Diffraction</subject>
    <subject>Neutron source</subject>
    <subject schemeURI="http://purl.org/pan-science/PaNET/" subjectScheme="The Photon and Neutron Experimental Techniques Ontology" valueURI="http://purl.org/pan-science/PaNET/PaNET01217">neutron diffraction</subject>
  </subjects>
```

**Comment by kara-mela** [comment]
At DESY, we are currently extending the metadata use at different steps of the workflows. We consider to use PIDINST and also the PaNET terms. It would be beneficial to establish a direct connection on the instrument level, as the set of provided techniques is quite static and rarely undergoes updates. 

The PaNET has created a document with recommendations on how to [include their terms as metadata in DataCite](https://doi.org/10.5281/zenodo.14191575), which basically corresponds to the previous suggestions (except for some details).

```
<subject xml:lang="en" subjectScheme=”PaNET”
  schemeURI=”http://purl.org/pan-science/PaNET/PaNET.owl”
  valueURI=”http://purl.org/pan-science/PaNET/PaNET01135”
  classificationCode=”PaNET01135”
  >absorption spectroscopy</subject>
```

It would be great to have the `subject` property available, before we intensively start working with PIDINST, such that we can include the experimental terms right from the beginning.

**Comment by BaptisteCecconi** [comment]
By the way, since there is also an issue about keywords (#72), shall be have a dedicated metadata property for the measurement technique? 

I can see applications where we would need:
- `subjects`: such as *Radio astronomy* (i.e., the intended scientific domain keywords) 
- `MeasuredVariable`: such as *Spectral Flux Density*
- `ExperimentalTechnique`: such as *Heterodyne radio receiver*

By the way, I would like to use this `ExperimentalTechnique` metadata to provide several keywords related to the instrument. In my example, I may want to provide details, for discovery purposes, as: 
- `ExperimentalTechnique`: *Heterodyne radio receiver*, *Short dipole antenna*, *Automatic gain control*, *Polyphase filtering*, etc.

*NB*: all terms in italic should be linked to ontologies (but I didn't lookup for such terms, at this point)

--- 

I would rather use a term like `InstrumentalTechnique` or `MeasurementTechnique`, because `ExperimentalTechnique` may not be applicable to all cases. However, that latter comment is a minor issue.

**Comment by paulmillar** [comment]
Just to add my 2c-worth,

Personally, I would favour using a different name than `subjects`: that name is overloaded (already used in DataCite with slightly different semantics) and (personally) I think "subject" carries a different meaning.  You mentioned "intended scientific domain" when describing the property, so how about calling it `ScientificDomain` instead?  The property should accept multiple values (an instrument might support multiple scientific domains) but the property name should be singular in order to be consistent with the existing fields.

I believe there are some existing ontologies/taxonomies for scientific domains.  While PIDINST shouldn't enforcing any particular taxonomy, providing some examples might encourage reuse of an existing taxonomy (rather than developing an unnecessary new taxonomy or just using plain text), and so improve interoperability.

For `ExperimentalTechnique`, one complication (one that we're working through in a different context) is this can be multi-layered.  In rather abstract terms, an instrument might have multiple "modes of operation" (just to give it a neutral name).  These modes of operation might operate exclusively (the instrument supports only one at a time)  or the instrument might allow multiple modes concurrently.  Within one of these modes of operation, the instrument might have multiple terms that describe different aspects of that mode of operation (`Heterodyne radio receiver`, `Short dipole antenna`, `Automatic gain control`, `Polyphase filtering` in your example).

In PaNET, an individual term corresponds to a mode of operation.  The different aspects of that mode of operation are captured within PaNET itself, linked to from the term.  This has the advantage of any experimental technique requiring only a single term (without loosing information), but it also create a potential combinatorial explosion and makes it harder to describe new techniques in some cases.

You can find more details on the conceptual model of PaNET in [the documentation](https://expands-eu.github.io/ExPaNDS-experimental-techniques-ontology/).

Personally, on the `InstrumentalTechnique` vs `MeasurementTechnique` vs `ExperimentalTechnique` question. I don't think the choice doesn't matter too much.  Any preference will likely be scientific-domain-specific: in PaNOSC, we happen to use "experimental technique".  I think `MeasurementTechnique` would also be OK.  `InstrumentTechnique` might cause a moment of confusion, but would be quickly understood.

**Comment by RKrahl** [comment]
I agree that we should not confuse this property with keywords or subjects.  Maybe my initial comment was misleading: I meant to say: “look what's posible in DataCite, we want that as well”. I didn't meant to suggest copying the DataCite `Subject` property over to PIDINST.  This proposal is not to be confused with #72.

My proposal was to have a property for the technique (lets use that neutral term for the moment) the instruments implements.  I would define the technique as: 

  “the protocol that the instrument follows or the physical phenomenon it makes use of in order make its observation.”

I believe this is significant enough for the description of an instrument to merit a dedicated property in the schema.  And yes, this would be a multi valued property, an instrument may implement more than one technique.  And of course, the property should allow and encourage linking to existing controlled vocabularies, so we'd need subproperties for adding the PID of the term in that vocabulary.

I don't think `ScientificDomain` would be appropriate here.  This is really something different.

Concerning the name of the property, I don't stick to `ExperimentalTechnique`.  I'm happy with `MeasurementTechnique`.  I'm not so sure about ` InstrumentTechnique`, because the technique exists independently of a given instrument and might be implemented in different ways.

**Comment by RKrahl** [comment]
NB: one argument in favour of `MeasurementTechnique` might be that this name hints that this property is complementary to `MeasuredVariable`.

**Comment by paulmillar** [comment]
Just for reference, for future work when updating the mapping to DataCite, there is some existing work that may be useful, here:

https://doi.org/10.5281/zenodo.14191575
